code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -103,3 +106,15 @@
 
 /obj/item/clothing/gloves/rapid/hug/attack_self(mob/user)
 	return FALSE
+
+/obj/item/clothing/gloves/thief
+	name = "thief's gloves"
```

Comment:
Shift clicks `Wait a second why are you wareing "Theifs Gloves"?

Reply:
yeah I was thinking of that, but I like my sprite ;w;
may just make them chameleon too